css/owned-card.styl

```diff
@@ -58,8 +58,28 @@
       background: lighten(COBOLT_BLUE, 20%)
 
 .collection-card
+  -webkit-border-radius: 1em
+  border-radius: 1em
+
   .name
     margin-bottom: 1em
+    height: 2.9em
+    overflow: hidden
+    text-overflow: ellipsis
+    display: -webkit-box
+    -webkit-box-orient: vertical
+    -webkit-line-clamp: 2
```

Comment:
fair enough. I grabbed this solution [here](http://stackoverflow.com/questions/3922739/limit-text-length-to-n-lines-using-css)

So what do you think I should do here? Trying to force a uniform height on the cards one way or another.... I could just set max height to 3 lines, but then there might be too much white space.... 

(are you saying don't bother trying to make it work cross-browser? or use the involved solution from the link)? or don't do this at all?)